backend/test/domains.test.ts

```diff
@@ -31,6 +32,7 @@ describe('domains', () => {
       isPassive: false
     }).save();
```

Comment:
do we need to add connection.close(); to this file too?

Reply:
Good catch. Missed it.